PyFlyt/core/drones/fixedwing XFLR5.py

```diff
@@ -0,0 +1,211 @@
+from __future__ import annotations
```

Comment:
What's the difference between this file and the other `fixedwing.py` file? Ideally, maintain only one. And rename this file to follow nospace conventions. Just `fixedwing.py` or `fixedwingXFLR5.py` is fine.

Reply:
Not needed anymore, stashed here as WIP initially, will remove